### PR TITLE
Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-ansible-role.git`](https://github.com/cisagov/skeleton-ansible-role.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

## ✅ Pre-approval checklist ##

Remove any of the following that do not apply. If you're unsure about
any of these, don't hesitate to ask. We're here to help!

- [x] All relevant type-of-change labels have been added.
- [x] All new and existing tests pass.

---

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

File: meta/main.yml [hunk 1]
```diff
@@ -17,21 +17,23 @@ galaxy_info:
   min_ansible_version: "2.10"
   namespace: cisagov
   platforms:
-    - name: Amazon Linux 2
-      versions:
-        - any
+    # Amazon Linux 2023 does not not currently offer ipa
```
Comment (jsf9k) [inline meta/main.yml]:
```suggestion
    # Amazon Linux 2023 does not not currently offer an IPA client package:
```

File: molecule/default/molecule-no-systemd.yml [hunk 2]
```diff
@@ -16,9 +16,11 @@ lint: |
   ansible-lint
   flake8
 platforms:
-  - image: amazonlinux:2
-    name: amazonlinux2
-    platform: amd64
+  # Amazon Linux 2023 does not currently offer ipa
```
Comment (jsf9k) [inline molecule/default/molecule-no-systemd.yml]:
```suggestion
  # Amazon Linux 2023 does not currently offer an IPA client package:
```

File: molecule/default/molecule-with-systemd.yml [hunk 3]
```diff
@@ -15,24 +15,17 @@ lint: |
...
-  - cgroupns_mode: host
-    command: /lib/systemd/systemd
-    image: geerlingguy/docker-debian9-ansible:latest
-    name: debian9-systemd
-    platform: amd64
-    pre_build_image: yes
-    privileged: yes
-    volumes:
-      - /sys/fs/cgroup:/sys/fs/cgroup:rw
+  # Amazon Linux 2023 does not currently offer ipa
```
Comment (jsf9k) [inline molecule/default/molecule-with-systemd.yml]:
```suggestion
  # Amazon Linux 2023 does not currently offer an IPA client package:
```

**Review by jsf9k — CHANGES_REQUESTED** [review]
I'm requesting some minor wording changes.

**Review by jsf9k — APPROVED** [review]

**Review by dav3r — APPROVED** [review]

**Pull request merged**